src/pages/user/authority/AuthorityCard.tsx

```diff
@@ -58,24 +56,45 @@ const AuthorityCard: React.FC<AuthorityCardProps> = ({ authority, isConnected =
 
   return (
     <StyledBoxContent isConnected={isConnected}>
-      <StyledAuthority>
+      <StyledContent>
         {!isConnected && <Radio color="primary" checked={isSelected} />}
-        {authority?.name}
-      </StyledAuthority>
-      <StyledPublicationContent>
+        <NormalText>{authority?.name}</NormalText>
+      </StyledContent>
+      <StyledContent>
         <StyledPublicationInfo>{t('authority.last_publication')}</StyledPublicationInfo>
         {isLoadingPublication ? (
           <CircularProgress />
         ) : publication?.title ? (
-          <Truncate lines={2} ellipsis={<span>...</span>}>
-            <NormalText>{publication.title}</NormalText>
-          </Truncate>
+          <NormalText>
+            <Truncate lines={2}>{publication.title}</Truncate>
```

Comment:
hvorfor tok du bort ellipsis-propen? Da vil jo teksten bare kuttes halvveis uten at det står `...` for å indikere at teksten er lengre?

Reply:
eller er det kanskje default `...` nå?

Reply:
det er default ja